services/user.js

```diff
@@ -40,13 +40,43 @@ class User {
     async grantToken(params, context){
         isLoggedInOrThrow(context);
         isValidUserStatus(context?.userInfo?.userStatus);
-        let accessToken = createToken(context?.userInfo, config.token_secret, config.tokenTimeout)
+        let accessToken = createToken(context?.userInfo, config.token_secret, config.tokenTimeout);
+        const aUser = await this.linkTokentoUser(context, accessToken);
```

Comment:
It doesn't look like we are using the "aUser" variable anywhere so I don't think a return value from linkTokenToUser is necessary.